yaml.properties

```diff
@@ -1,15 +1,21 @@
 category=Languages
 description=Allows the analysis of YAML documents
 homepageUrl=https://github.com/sbaudoin/sonar-yaml
-archivedVersions=1.5.2,1.6.0
-publicVersions=1.7.0
+archivedVersions=1.5.2,1.6.0,1.7.0
+publicVersions=1.8.0
 
 defaults.mavenGroupId=com.github.sbaudoin
 defaults.mavenArtifactId=sonar-yaml-plugin
 
 
+1.8.0.description=Brings support for SonarQube 9 LTS and 10; adds a rule and improves some others; fixes some issues
+1.8.0.sqVersions=[9.9.*,LATEST]
```

Comment:
It turns out you can't use a wildcard at the start of a range. This needs to start with `9.9`